frontend/pages/policies/ManagePoliciesPage/_styles.scss

```diff
@@ -295,4 +295,8 @@
       }
     }
   }
+
+  .component__last-updated-text {
+    font-weight: normal;
+  }
```

Comment:
Why normal here and bold everywhere else? e.g.

<img width="288" alt="image" src="https://github.com/user-attachments/assets/efbe0415-cd76-49a6-a55b-850b156bbbef" />

Reply:
I updated globally here after confirming with @eugkuo  https://github.com/fleetdm/fleet/pull/25143/files#r1904469843

Reply:
@eugkuo 's comment to be consistent and unbold everywhere: https://github.com/fleetdm/fleet/pull/25143#issuecomment-2573427272

Reply:
Sounds good, @jacobshandling in that case let's remove this carve-out so that it uses the global style

Reply:
@RachelElysia I'm going to merge this in since it's approved – since your PR is still waiting for review, can you please remove this one-off style in favor of the global update in your PR?